client/src/App.jsx

```diff
@@ -8,14 +8,34 @@ import SettingsPage from '../src/pages/SettingsPage';
 import FavoritePage from '../src/pages/FavoritePage';
 import DetailedGasStationPage from '../src/pages/DetailedGasStationPage';
 
+const stationsApiKey = process.env.REACT_APP_STATIONS_API_KEY;
+
 function App() {
+  const [stations, setStations] = useState([]);
+  const [radius, setRadius] = useState(5);
+  const [apiRadius, setApiRadius] = useState(5000);
+  const [latitude, setLatitude] = useState(null);
+  const [longitude, setLongitude] = useState(null);
+  const [position, setPosition] = useState(null);
   const [fuelValue, setFuelValue] = useState('e5');
   const [favoriteIDs, setFavoriteIDs] = useState(
     () => JSON.parse(localStorage.getItem('favoriteIDs')) ?? []
   );
 
+  const url = `https://api.tankentanken.de/gas-stations?latitude=${latitude}&longitude=${longitude}&radius=${apiRadius}`;
   const FuelInfo = fuelValue.charAt(0).toUpperCase() + fuelValue.slice(1);
 
+  //Calculate radius for API
+  function calculateApiRadius(radius) {
```

Comment:
I think as name `calculateRadius` would be fine as this function could also be used for other purposes. :)